ci/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM gradle:jdk17-jammy
+COPY ./install_software.sh .
+RUN sh install_software.sh
```

Comment:
would suggest placing the install command inside de docker file. makes it easy to see that is happening and allows for docker caching and multistage builds (https://docs.docker.com/build/building/multi-stage/). Installing docker in docker in maybe not the best option en general. best experience i had was using https://github.com/GoogleContainerTools/kaniko ( already used by molgenis)

Reply:
> installing docker in docker in maybe not the best option en general
In the current circle ci we also run docker in docker. I don't see reason to change that now.